src/main/resources/sql/db_sql.sql

```diff
@@ -0,0 +1,32 @@
+drop table vesta_schema.user;
+drop table vesta_schema.user_role;
+drop table vesta_schema.role;
+
+create table vesta_schema.user(
+    id serial primary key,
+    name varchar (50) UNIQUE not null ,
```

Comment:
what you mean by name? username or first name? if first name then why it need to be UNIQUE